test/js_test/generate.sh

```diff
@@ -3,7 +3,7 @@
 rm -rf generated/
 mkdir -p generated/
 
-protoc \
+grpc_tools_node_protoc \
```

Comment:
Possibly a silly question, but why did this need to change? Does the usage example in the README need to be updated as well?